lib/sms_autofill.dart

```diff
@@ -8,9 +8,9 @@ import 'package:pin_input_text_field/pin_input_text_field.dart';
 export 'package:pin_input_text_field/pin_input_text_field.dart';
 
 class SmsAutoFill {
-  static SmsAutoFill _singleton;
+  static SmsAutoFill? _singleton;
```

Comment:
Probably `late` is enough here too instead of nullable

Reply:
This. Make this not null.